### DISCUSSION TITLE
ChatGPT continue to seem not working...

### DISCUSSION THREAD

**Discussion by Ale111** [llm-workflow-engine/llm-workflow-engine#85] [General]
Hi all,
I've just downloaded the last version of the wrapper, unfortunately this is what I see on the terminal:

```
Solid:chatgpt_wrapper ale$ python chatgpt.py -l botlog.txt
Provide a prompt for ChatGPT, or type !help or ? to list commands.
1> !session

```

... and no more!

Same for any prompt.

Seem that OpenAI has lately changed something on backend-api. 
Up to 12 hours ago everything worked fine with the previous version.

Any update?

Thank you.

**Comment by Tecuya** [comment]
Hmm I just got it to work for me.  Make sure you are logged in and on version 0.3.11

**Reply by Ale111** [reply]
Hi, thanks for the reply,

unfortunally it does not work :( ...  I have reinstalled the last version without changes from before.


```
Solid:chatgpt ale$ pip install git+https://github.com/mmabrouk/chatgpt-wrapper
...
...
...
Running setup.py install for chatGPT ... done
Successfully installed chatGPT-0.3.11

Solid:chatgpt ale$ chatgpt -l botlog.txt 
Provide a prompt for ChatGPT, or type !help or ? to list commands.
1> ?

Documented commands (type !help <topic>):
=========================================
!ask  !context  !exit  !file  !help  !log  !nav  !new  !read  !session  !stream

1> !session



```

... and no more!

Any suggestions?

Thanks.

**Reply by free-as-in-freedom** [reply]
I am a bit worried because for the first time today I got a captcha on chatgpt. could this be the issue?

**Reply by Ale111** [reply]
> I am a bit worried because for the first time today I got a captcha on chatgpt. could this be the issue?

it could be a cause.

**Comment by Ale111** [comment]
It would seem that backend API has returned to work today... Great!

```
Solid:chatgpt3 ale$ chatgpt
Provide a prompt for ChatGPT, or type !help or ? to list commands.
1> ?

Documented commands (type !help <topic>):
=========================================
!ask  !context  !exit  !file  !help  !log  !nav  !new  !read  !session  !stream

1> !session

 • Session information refreshed.  The session appears to be usable.                                                                                                        

1> !log botlog.txt

 • Logging enabled, appending to 'botlog.txt'.  
...

```

Thank you for the update.
